controllers/base.go

```diff
@@ -71,6 +71,18 @@ func (c *ApiController) ReadRequestFromQueryParams() BaseDataManageRequest {
 		Organization: c.Input().Get("organization"),
 	}
 
+	if !util.FilterField(result.Field) {
+		result.Field = ""
```

Comment:
should we clean up value also in this case?

Reply:
yes, done.